### ISSUE TITLE
Unable to install software

### ISSUE THREAD

**Issue by Kptainflintt** [wh1te909/tacticalrmm#621]
**Server Info (please complete the following information):**
 - OS: Debian 10
 - Browser: Firefox
 - RMM Version (as shown in top left of web UI): 0.7.2

**Installation Method:**
  - [ X] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): 
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2]

**Describe the bug**
When trying to install any software via rmm portal, getting this output : 

exec: "choco.exe": executable file not found in %PATH%: 

**To Reproduce**
1. Go to rmm portal
2. Select any computer
3. Add a software

**Expected behavior**
A way to add choco.exe to path in order to install softwares remotly
Tried to add manually on a host with SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin" but no success

**Comment by silversword411** [comment]
You restart the agent? Sometimes path doesn't update in a timely manner

**Comment by Kptainflintt** [comment]
Hello,

Sorry for the late reply.
Yes, I have restarted the agent and even the posts. If I log in with a user and open a command prompt, the "choco" command is recognized but nothing from the portal (the agent must use the SYSTEM user I guess).

I can't test today, portal is buggy, I cannot connect to any agents, even witch a remote background (blank page). I'll try to reboot server and come back to give news.

Many bugs since the update to 0.7.2 ... 

Thank you

**Comment by silversword411** [comment]
Sorry, restart the computer the agent is running on. I should have been more clear. You can also validate choco is working on a computer. What agent OS/build are you running on? You didn't include it in the issue description. 

I haven't see or heard of any bugs with 0.7.2, if you think something is broken/not working create an issue with testing process and it'll be checked.

You can verify choco using Remote Background

![2021-07-08_103442 - check choco as system](https://user-images.githubusercontent.com/13395348/124940862-55c78080-dfd8-11eb-986f-b469f0281ab8.png)

**Comment by Kptainflintt** [comment]
Hi,

Sorry for OS and agent version!

Agent version was 1.5.8, OS is W10 Education.

I have checked already "choco" command via remote background with the same result. However, I noticied that auto-update are not functionning so I've launched manual update (to 1.5.9) and remote install work now on this computer.

Same thing with another PC : choco working from command line with an user logged in but not in remote background. Agent manually updated resolve the issue.

Thank you, and it's noted for other bugs. I'll try to recod all steps to reproduce.

Have a nice day
